datadog_checks_dev/tests/tooling/manifest_validator/input_constants.py

```diff
@@ -196,11 +198,208 @@
 
 DISPLAY_ON_PUBLIC_VALID_MANIFEST = JSONDict({"display_on_public_website": True})
 
-INVALID_DIFF = 'app_id id'
+IMMUTABLE_ATTRIBUTES_V1_MANIFEST = {"manifest_version": "1.0.0"}
+
+IMMUTABLE_ATTRIBUTES_V2_MANIFEST = JSONDict({"manifest_version": "2.0.0"})
+
+IMMUTABLE_ATTRIBUTES_PREV_MANIFEST_INVALID = json.dumps(
+    {
+        "app_id": "datadog-oracle",
+        "assets": {
+            "dashboards": {"oracle": "https://app.datadoghq.com/screen/integration/240/oracle-database---overview"},
+            "monitors": {"cool-monitor": "link"},
+            "integration": {
+                "configuration": {"spec": "assets/configuration/spec.yaml"},
+                "events": {"creates_events": True},
+                "id": "oracle",
+                "metrics": {
+                    "auto_install": True,
+                    "check": "oracle.session_count",
+                    "metadata_path": "metrics_metadata.csv",
+                    "prefix": "oracle.",
+                },
+                "service_checks": {"metadata_path": "assets/service_checks.json"},
+                "source_type_name": "Oracle Database",
+            },
+        },
+        "author": {
+            "homepage": "https://www.datadoghq.com",
+            "name": "Datadog",
+            "sales_email": "help@datadoghq.com",
+            "support_email": "help@datadoghq.com",
+        },
+        "display_on_public_website": True,
+        "legal_terms": {},
+        "manifest_version": "2.0.0",
+        "oauth": {},
+        "pricing": [{"billing_type": "free"}],
```

Comment:
I think we've resigned to making this 
```suggestion
        "pricing": [],
```
cc @mgarabed I'm not quite certain if we went with an empty list or an omission of the field?

Reply:
Ignore me here. Confirmed with Mike that we settled on `free` as you have it